client/state/plans/selectors/get-plan-prices.ts

```diff
@@ -26,6 +32,19 @@ export function getPlanPrices(
 ): PlanPrices {
 	const plan = getPlan( planSlug );
 	const productId = plan?.getProductId();
+	const currentSitePlan = getCurrentPlan( state, siteId );
+
+	if ( currentSitePlan?.productSlug === planSlug ) {
+		return {
+			rawPrice:
+				( siteId &&
```

Comment:
`Non-blocking`:

Totally not mandatory, but how do we feel about scooting this set of conditionals out of the object literal to make it easier to read?

```
	const rawPrice =
		( siteId &&
			planSlug === currentSitePlan?.productSlug &&
			getSitePlanRawPrice( state, siteId, planSlug, { returnMonthly } ) ) ||
		0;

	if ( currentSitePlan?.productSlug === planSlug ) {
		return {
			rawPrice,
			discountedRawPrice: 0,
			planDiscountedRawPrice: 0,
		};
	}
```